pkg/jsonnet/imports.go

```diff
@@ -92,7 +92,7 @@ func importRecursive(list map[string]bool, vm *jsonnet.VM, node ast.Node, curren
 
 		contents, foundAt, err := vm.ImportAST(currentPath, p)
 		if err != nil {
-			return errors.Wrap(err, "importing jsonnet")
+			return errors.Wrapf(err, "importing %s from %s", p, currentPath)
```

Comment:
```suggestion
			return fmt.Errorf("importing '%s' from '%s': %w", p, currentPath, err)
```

Newer Go versions ship `%w` for stdlib-native error wrapping, we use it in other places so let's do here as well.

Actual feedback: Let's wrap the paths in single quotes, so you more easily realize when an argument might be an empty string